beacontools/scanner.py

```diff
@@ -229,6 +318,55 @@ def process_packet(self, pkt):
                     self.callback(bt_addr, rssi, packet, properties)
                     return
 
+    def process_ext_packet(self, pkt):
```

Comment:
Same here as for `set_ext_scan_parameters`. Please merge this into `process_packet` to avoid duplicate code. The only thing that seems to be different is the offset of `payload` and `rssi`.